test/python/test_ibmqjob.py

```diff
@@ -0,0 +1,143 @@
+        quantum_job = QuantumJob(qobj, backend, shots=1024, preformatted=True)
+        job = backend.run(quantum_job)
+        result = job.result()
+        counts_qx = result.get_counts(result.get_names()[0])
+        counts_ex = {'00': 512, '11': 512}
+        states = counts_qx.keys() | counts_ex.keys()
+        # contingency table
+        ctable = numpy.array([[counts_qx.get(key, 0) for key in states],
+                              [counts_ex.get(key, 0) for key in states]])
+        contingency = chi2_contingency(ctable)
+        self.log.info('chi2_contingency: %s', str(contingency))
+        self.assertGreater(contingency[1], 0.01)
+
+    @slow_test
+    def test_run_async_simulator(self):
+        backend = self._provider.available_backends({'simulator': True})[0]
+        self.log.info('submitting to backend %s', backend.name)
+        num_qubits = 5
+        qr = QuantumRegister(num_qubits, 'qr')
+        cr = ClassicalRegister(num_qubits, 'cr')
+        qc = QuantumCircuit(qr, cr)
+        for i in range(num_qubits-1):
+            qc.cx(qr[i], qr[i+1])
+        qc.measure(qr, cr)
+        qobj = qiskit._compiler.compile([qc]*10, backend)
+        quantum_job = QuantumJob(qobj, backend, shots=1e3, preformatted=True)
+        num_jobs = 5
+        job_array = [backend.run(quantum_job) for _ in range(num_jobs)]
+        time.sleep(0.1)  # give time for jobs to start (better way?)
+        num_queued = sum([job.queued for job in job_array])
+        num_running = sum([job.running for job in job_array])
+        self.log.info('number of currently queued jobs: %d/%d',
+                      num_queued, num_jobs)
+        self.log.info('number of currently running jobs: %d/%d',
+                      num_running, num_jobs)
+        self.assertTrue(all([(job.running or job.queued) for job in job_array]))
+
+    @slow_test
+    def test_run_async_device(self):
+        backend = self._provider.available_backends({'simulator': False})[0]
+        self.log.info('submitting to backend %s', backend.name)
+        num_qubits = 5
+        qr = QuantumRegister(num_qubits, 'qr')
+        cr = ClassicalRegister(num_qubits, 'cr')
+        qc = QuantumCircuit(qr, cr)
+        for i in range(num_qubits-1):
+            qc.cx(qr[i], qr[i+1])
+        qc.measure(qr, cr)
+        qobj = qiskit._compiler.compile(qc, backend)
+        quantum_job = QuantumJob(qobj, backend, shots=1e5, preformatted=True)
+        num_jobs = 3
+        job_array = [backend.run(quantum_job) for _ in range(num_jobs)]
+        num_queued = sum([job.queued for job in job_array])
+        num_running = sum([job.running for job in job_array])
+        self.log.info('number of currently queued jobs: %d/%d',
+                      num_queued, num_jobs)
+        self.log.info('number of currently running jobs: %d/%d',
+                      num_running, num_jobs)
+        self.assertTrue(all([(job.running or job.queued) for job in job_array]))
```

Comment:
Can you also expand this line in the same way as the note in `test_run_async_simulator()`?